README.md

```diff
@@ -52,39 +52,30 @@ gem install --install-dir vendor/bundle/ruby/<ruby_version_numbwe> nokogiri -v "
  1. Fork this repository on Github.
  2. Clone *your forked repository* with `git clone https://github.com/YOURUSERNAME/dev-center.git`
  3. `cd dev-center`
- 4. Install all dependencies: `bundle install`. If you are having trouble with some of the gems, try running `bundle update`, then run `bundle install` again.  If `bundle` is not available, `gem install bundler`.
- 5. Run `git submodule init`, initializing the submodule for our UI toolkit, Synapse. **Do bear in mind that you will need to able to communicate with [GitHub via SSH](https://help.github.com/en/articles/connecting-to-github-with-ssh) for this to function as expected.**
- 6. Run `git submodule update`, cloning down submodule code.
- 7. To build Synapse dist files, run `cd synapse && npm ci && npm run build && cd ..`
- 8. Start the local server:
+ 5. To set up submodules and install dependencies, run `npm run setup`.
```

Comment:
Can we add a note about making sure that `rvm` is installed? I got an error when running `npm run setup` because I didn't have `nvm` on my machine. https://rvm.io/rvm/install

Reply:
Added!